app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluation.tsx

```
@@ -1211,50 +1288,141 @@ export default function Evaluation(props: EvaluationProps) {
             <TextField
-              defaultValue={barConfigDialogState.limit}
+              defaultValue={classPerformanceDialogConfig.limit}
               size="small"
               type="number"
               onChange={(e) => {
-                setBarConfigDialogState((state) => ({
+                const newLimit = parseInt(e.target.value);
+                setClassPerformanceDialogConfig((state) => {
+                  return {
+                    ...state,
+                    limit: isNaN(newLimit) ? undefined : newLimit,
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)